Build/15.0/packages.config

```diff
@@ -68,5 +68,5 @@
   <package id="Microsoft.VisualStudio.Editor.Implementation" version="15.8.525" />
   <package id="Microsoft.VisualStudio.Platform.VSEditor" version="15.8.525" />
   <package id="Microsoft.VisualStudio.Text.Internal" version="15.8.525" />
-  <package id="Microsoft.VisualStudio.Python.LanguageServer" version="0.1.28" />
+  <package id="Microsoft.VisualStudio.Python.LanguageServer" version="0.1.29" />
```

Comment:
Since the updated language server is going into 16.0, why update 15.0?

Reply:
While this file exists, I'd rather keep it in sync. Not ready to delete the file yet, would rather wait until 16.0 is out.